sdk/testing/src/test/java/io/opentelemetry/sdk/testing/exporter/InMemorySpanExporterTest.java

```diff
@@ -23,13 +23,17 @@
 
 /** Unit tests for {@link InMemorySpanExporter}. */
 class InMemorySpanExporterTest {
-  private final SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder().build();
-  private final Tracer tracer = sdkTracerProvider.get("InMemorySpanExporterTest");
   private final InMemorySpanExporter exporter = InMemorySpanExporter.create();
 
+  private Tracer tracer;
+
   @BeforeEach
   void setup() {
-    sdkTracerProvider.addSpanProcessor(SimpleSpanProcessor.builder(exporter).build());
+    tracer =
+        SdkTracerProvider.builder()
```

Comment:
I don't think Simple processor needs this, but it is a good pattern to follow and shutdown the ask all the time.